src/banking-client.ts

```diff
@@ -74,7 +74,7 @@ export class BankingService {
   // Accepts the sourceAccount (string), amount (number), and an referenceId (string) for idempotent transaction tracking.
   // Returns a transaction id when successful
   // Returns various errors based on amount and account number.
-  withdraw(sourceAccount: string, amount: number, referenceId: string): string {
+  async withdraw(sourceAccount: string, amount: number, referenceId: string): Promise<string> {
```

Comment:
The activities assume that these methods are async but are not implemented as such so TS was showing errors